ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -24,6 +24,8 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
                             bool enable_mpi_threads);
 
 static int param_priority;
+static char *prov_include;
+static char *prov_exclude;
```

Comment:
This is done below (new line 75) in the same manner as param_priority.
Do you want me to move it to the declaration line?